src/app/extensions/quickorder/shared/header-quickorder/header-quickorder.component.html

```diff
@@ -1,4 +1,6 @@
-<div class="d-none d-md-block quick-order-links">
-  <fa-icon class="mr-2 text-primary align-middle" [icon]="['fas', 'fast-forward']"></fa-icon>
-  <a rel="quick-order" routerLink="/quick-order">{{ 'quickorder.page.link' | translate }}</a>
-</div>
+<fa-icon
+  class="mr-2 align-middle"
```

Comment:
Maybe we could remove the usage of Bootstrap Utility Classes again? This is a paradigm change that would make customizations different.